### ISSUE TITLE
Authentication Guide

### ISSUE THREAD

**Issue by dannnnthemannnn** [matthewdean/roblox-web-apis#41]
Is there any guide for authenticating for these apis? Are these endpoints to be used with tokens generated using roblox cloud? https://create.roblox.com/docs/cloud/open-cloud/oauth2-overview

**Comment by OverlordShin** [comment]
Yes, you can see [here](https://create.roblox.com/docs/en-us/cloud/open-cloud/api-keys) on how to get a key. As for a guide to authenticate an Oauth app you can see [here](https://create.roblox.com/docs/en-us/cloud/open-cloud/oauth2-sample). If you mean endpoints such as inventory V2, see [here](https://create.roblox.com/docs/en-us/cloud/open-cloud/inventory).

**Comment by matthewdean** [comment]
Two separate schemes:

- **Legacy sites** (the README table): `.ROBLOSECURITY` cookie. Non-GET requests also need an `x-csrf-token` header — send without one and the 403 response hands you the token to retry with. Unofficial and unsupported.
- **Open Cloud** (`apis.roblox.com`): API keys or OAuth 2.0 — the supported path, and what you linked. https://create.roblox.com/docs/cloud

The README now has an Open Cloud section covering this. Closing — thanks @OverlordShin for the original answer.
